charts/nextcloud/templates/cronjob.yaml

```
@@ -28,6 +28,9 @@ spec:
         {{- end }}
     spec:
       backoffLimit: {{ .backoffLimit }}
+      {{- if .activeDeadlineSeconds }}
```

Easier to read:
```suggestion
      {{- with .activeDeadlineSeconds }}
      activeDeadlineSeconds: {{ . }}
```
PS: Mobile i could not select the other line.